support-frontend/assets/helpers/abTests/landingPageAbTests.ts

```diff
@@ -68,14 +68,25 @@ export const fallBackLandingPageSelection: LandingPageVariant = {
 			},
 			label: { copy: 'Recommended' },
 		},
-		TierThree: {
-			title: ProductTierLabel.TierThree,
+		DigitalSubscription: {
+			title: ProductTierLabel.DigitalSubscription,
 			benefits: [
```

Comment:
perhaps this is for a follow up task, but I wonder if these hardcoded fallbacks could reference the copy in productCatalog.ts